test/features/step_definitions/authentication.steps.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Copyright 2017 Longtail Ad Solutions Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ **/
+
+/* jshint esversion: 6 */
+/* globals require, browser, angular, expect, $ */
+
+const
+    {defineSupportCode} = require('cucumber');
+
+defineSupportCode(function ({Given, When, Then}) {
+
+    Given('the login modal is visible', function () {
+        let element = $('.jw-popup-login');
+        return expect(element.isDisplayed()).to.eventually.equal(true);
```

Comment:
There's no need for the assignment.


```js
return expect($('.jw-popup-login').isDisplayed()).to.eventually.equal(true);
```

Either that, or add a newline before the return. Apply this everywhere.